package.json

```diff
@@ -8,7 +8,7 @@
     "url": "https://project-git-repo.tld"
   },
   "scripts": {
-    "prepare": "lerna bootstrap --hoist",
+    "prepare": "lerna bootstrap --hoist && husky install",
```

Comment:
Is it really necessary to manually call `husky install`? I thought husky would automatically install himself.

Reply:
Husky dropped their auto-install. They now recommend that you use the install command on prepare: https://blog.typicode.com/husky-git-hooks-autoinstall/